tests/90-mpi.bats

```diff
@@ -0,0 +1,12 @@
+#!/bin/npx bats
+
+load "../node_modules/bats-support/load"
+load "../node_modules/bats-assert/load"
+. "tests/utils.sh"
+
+
+@test "test MPI is installed" {
+	run juju run-action -m "$JUJU_MODEL" slurmd/leader mpi-install --wait
+	run juju run --unit slurmd/leader --model "$JUJU_MODEL" "PATH=/usr/lib64/mpich-3.2/bin:$PATH && mpirun --version"
```

Comment:
Is the path the same on both OSes?

Why this is not part of `PATH` by default?

Reply:
On Ubuntu, it is part of `PATH` by default, so this is not needed. But on CentOS, the mpich is installed as a module.

So on CentOS, we need to load the module:

`module load mpi/mpich-3.2-x86_64`

In order to set the `PATH`.

This is done in the` /etc/bashrc`, but the `juju run --unit slurmd/leader` command does not execute `/etc/bashrc`.

Thus, the alternative is to set `PATH` when running the command.

Reply:
The mpich path is the same in all CentOS 7 installations

Reply:
What module system is used? How it is installed? How it is configured? Is it compatible with the systems provided by the other actions we have? Is the same installation method for both OSes?

Are you adding mpich as a module, and automatically loading it when users log in if they use Bash?

Reply:
**What module system is used?** 

environment modules (http://modules.sourceforge.net/)

**How it is installed?** 

It is one of mpich dependencies.

So if it is not already available in the system, it is installed via package mananger (yum) along with mpich.

**How is it configured?** 

It does not require configuration. The package manager automatically creates the module when installing mpich.

**Is it compatible with the systems provided by the other actions we have?** 

Yes, it is.

**Is the same installation method for both OSes?**

Ubuntu does not install the environment modules as a dependency for mpich. It installs mpich and that is it. Once installed mpich is available to use.

CentOS installs mpich (or any MPI implementation), and also installs environment modules. And automatically creates a module for mpich.

Thus, the step to load the mpich module is necessary only on CentOS.

**Are you adding mpich as a module, and automatically loading it when users log in if they use Bash?**

Yes. 
The command to load mpich is put in /etc/bashc, so mpich is loaded when a user log in.